src/App.js

```diff
@@ -1,23 +1,35 @@
 import React, { useState, useEffect } from 'react';
 import AddTodoForm from "./AddTodoForm";
 import TodoList from "./TodoList";
+//require('dotenv').config({ path: '.env.local' });
 function App() { 
-    const initialTodoList =  JSON.parse(localStorage.getItem('savedTodoList')) || []
+    const API_KEY="keyekdSu5UIFCvRi7"
+    const BASE_ID="appYHZzF0wZRM22FY"
```

Comment:
Oooh this isn't good!! You should keep these in a `.env.local` file. Read that documentation I sent you the other comment 👍